### PR TITLE
readme: example is broken with simple-get

### PR DESCRIPTION
**Pull request by reggi**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -65,11 +65,13 @@ module.exports = function fetch (callback) {
 ```
 
 ```js
-var proxyquire = require('proxyquire');
-var fetch = proxyquire('./get', {
+var assert = require('assert');
+var proxyquire = require('proxyquire').noCallThru();
```
Comment (bendrucker) [inline README.md]: Undo this change and require proxyquire first

File: README.md [hunk 2]
```diff
@@ -65,11 +65,13 @@ module.exports = function fetch (callback) {
...
+
+var fetch = proxyquire('./use', {
```
Comment (bendrucker) [inline README.md]: Don't change the module name

File: README.md [hunk 3]
```diff
@@ -65,11 +65,13 @@ module.exports = function fetch (callback) {
...
   'simple-get': function (url, callback) {
     process.nextTick(function () {
-      callback(null, fakeResponse)
+      callback(null, {'statusCode': 200})
```
Comment (bendrucker) [inline README.md]: Don't quote keys unnecessarily

File: README.md (hunk 1 above)
Comment (reggi) [reply README.md]: I do think that adding `noCallThru` is really essential because I didn't have `simple-get` installed and it was failing and I had no clue why.

**Comment by thlorenz** [comment]
I personally actually like `fakeResponse` more for the example.
It communicates much clearer that we are mocking things here even though you cannot run the example since that var doesn't exist.
Another option is to just assign `fakeResponse` in a line above, i.e. `const fakeResponse = { status: 200 }`, that way the example runs + clearly communicates the point.

At any rate please fixe the spacing to `{ status: 200 }` in case we go with the change anyways.
